src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -389,17 +389,34 @@ impl MDARegions {
         for mda in self.mda_headers.iter() {
             assert!(mda
                 .as_ref()
-                .map_or_else(|| true, |mda| mda.used != Bytes(0)));
+                .map_or_else(|| true, |mda| mda.used.bytes() != Bytes(0)));
         }
     }
 }
 
+/// A type representing the actual size of variable length metadata written within this metadata
+/// region. This amount must never be greater than the size of the region allocated for the variable
+/// length metadata, which has `MDADataSize` type.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub struct MetaDataSize(Bytes);
+
+impl MetaDataSize {
+    pub fn new(used: Bytes) -> MetaDataSize {
+        MetaDataSize(used)
+    }
+
+    pub fn bytes(&self) -> Bytes {
```

Comment:
There was a travis-ci test failure:
```
error: this argument is passed by reference, but would be more efficient if passed by value
   --> src/engine/strat_engine/backstore/metadata/mda.rs:408:18
    |
408 |     pub fn bytes(&self) -> Bytes {
    |                  ^^^^^ help: consider passing by value instead: `self`
    |
    = note: `-D clippy::trivially-copy-pass-by-ref` implied by `-D warnings`
```

Reply:
@GuillaumeGomez We have a clippy make target, which you can run on your local.